blueprints/networking/psc-hybrid/main.tf

```diff
@@ -0,0 +1,110 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  project_id         = var.project_create ? module.project.project_id : var.project_id
+  vpc_producer_id    = var.vpc_create ? module.vpc_producer.network.id : var.vpc_config["producer"]["id"]
```

Comment:
can you break long lines? eg

```hcl
vpc_producer_main  = (
  var.vpc_create
  ? module.vpc_producer.subnets["${var.region}/${var.prefix}-main"].id
  : var.vpc_config["producer"]["subnet_main_id"]
)
```

When in doubt, please use the contributing guide as a reference.